application/views/admin/super/footer.php

```diff
@@ -52,7 +53,7 @@ class='btn btn-warning btn-sm'
         <div class="modal-content">
             <div class="modal-header">
                 <button type="button" class="close" data-dismiss="modal">&times;</button>
-                <h4 class="modal-title"><?php eT("Confirm"); ?></h4>
+                <h3 class="modal-title"><?php eT("Confirm"); ?></h3>
```

Comment:
I'm upset about h1/h2 etc ... Did modal need a hX ? Or to be a `<article>` with h1 :(